oio/admin/agent.py

```diff
@@ -0,0 +1,570 @@
+                if opts.get(field):
+                    job['config'][field] = opts.get(field)
+
+            bases_all = self.chunk_bases(bases, int(
+                job['config'].get('concurrency', '1')))
+
+            for bases in bases_all:
+                job["config"]["bases"] = bases
+                job["processes"].append(mp.Process(
+                    target=self.move_meta2,
+                    args=(
+                        job["config"],
+                        job["stats"],
+                        job["control"],
+                    )
+                ))
+            job['stats']['total'] = len(bases)
+
+        elif type_ == "rawx" and action == "move":
+            try:
+                excluded = self.excluded("rawx", opts.get('exclude', []))
+            except Exception as e:
+                err = "Could not parse exclusion list: %s" % format_exc(e)
+                return None, err
+
+            job['config'] = dict(
+                src=src,
+                volume=vol,
+                namespace=self.namespace,
+            )
+            for field in [("bps", "bytes_per_second"),
+                          ("cps", "chunks_per_second"),
+                          ("concurrency", "concurrency"),
+                          ("target", "usage_target"),
+                          ("minsize", "min_chunk_size"),
+                          ("maxsize", "max_chunk_size")]:
+                if opts.get(field[0]):
+                    job["config"][field[1]] = opts[field[0]]
+            if excluded:
+                job["config"]["excluded_rawx"] = ",".join(excluded)
+
+            job["processes"].append(mp.Process(
+                target=self.move_blobs,
+                args=(
+                    job['config'],
+                    job["stats"],
+                    job["control"]
+                )
+            ))
+        elif type_ == "rawx" and action == "rebuild":
+            job['config'] = dict(
+                src=src,
+                volume=vol,
+                namespace=self.namespace,
+                report_interval=0,
+            )
+
+            job['stats']['total'] = mp.Value('i')
+
+            # TODO: VDO: support rdir_fetch_limit, rdir_timeout, retry_delay
+            job["processes"].append(mp.Process(
+                target=self.rebuild_rawx,
+                args=(
+                    job['config'],
+                    job["stats"],
+                    job["control"]
+                )
+            ))
+        else:
+            return None, "Unknown job %s %s" % (type_, action)
+
+        for p in job['processes']:
+            p.start()
+        self.jobs[id] = job
+        return id, None
+
+
+class BaseAdminAgentHandler(BaseHTTPRequestHandler):
+    """
+    Mover agent handler
+    Handles incoming HTTP requests
+    """
+    agent = None
+
+    def __init__(self, *args, **kwargs):
+        BaseHTTPRequestHandler.__init__(self, *args, **kwargs)
```

Comment:
No need to declare `__init__` if the only statement is calling the superclass' constructor.
BTW, prefer calling `super(BaseAdminAgentHandler, self).__init__(*args, **kwargs)`.